setup/features/server-group-launch-settings/aws-ec2/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: single
+title:  "Overview"
+sidebar:
+  nav: setup
+---
+
+## Overview
+An AWS EC2 Server Group offers a number of launch setting configurations that can enhance your experience.
+
+### Launch Settings
+An AWS Server Group can be set up with two types of launch settings. However, <b>you must enable Launch Templates support for all of your applications if you want to use the latest AWS features.
```

Comment:
What are the two types of launch settings? It would be clearer to mention them. "You can set up an AWS Server Group with either a launch template or a launch configuration."